concatenate_flows/Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+coredatamodules = {editable = true, ref = "v0.6.2", git = "https://www.github.com/AfricasVoices/CoreDataModules"}
```

Comment:
I strongly recommend switching to v0.7.0, but please check the breaking changes in the release notes before you do.